src/componenets/TextForm.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+class TextForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: ""
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
same for these two methods. Use arrow functions instead of explicitly binding this here in constructor